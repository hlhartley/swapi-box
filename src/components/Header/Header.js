import React from 'react';
import '../../main.scss';

const Header = () => {
    return (
        <div className='header'>
            <h1 className='header-text'>SWAPI-box</h1>
        </div>
    )
}

export default Header;