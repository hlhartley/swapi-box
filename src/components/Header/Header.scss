.header {
    display: flex;
    flex-flow: column;
    justify-content: center;
    align-items: center;
    color: black;
    height: 250px;
    background: url(images/r2d2-image.jpg);
    background-repeat: no-repeat;
    background-size: 100% 320px;
}

.header-text {
    font-family: 'Play', sans-serif;
    font-size: 71px;
    margin-left: -220px;
    margin-top: 150px;
    color: #011c44;
    text-shadow: 2px 2px 5px #aea79f
}