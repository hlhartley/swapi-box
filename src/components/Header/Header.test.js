import React from 'react';
import ReactDOM from 'react-dom';
import Header from './Header';
import { shallow } from 'enzyme';

let wrapper;
describe('Header', () => {

  beforeEach(() => {
    wrapper = shallow(<Header/>); 
  })

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Header />, div);
    ReactDOM.unmountComponentAtNode(div);
})

  it('should exist', () => {   
    expect(wrapper).toBeDefined()
  })

  it('should match the snapshot with all data passed in correctly', () => {
    expect(wrapper).toMatchSnapshot();
  })

  // it.skip('should have a proper default state', () => {
  //   expect(wrapper.state()).toEqual({
      
  //   })
  })
