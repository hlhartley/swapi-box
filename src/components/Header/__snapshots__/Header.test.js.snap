// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot with all data passed in correctly 1`] = `
<div
  className="header"
>
  <h1
    className="header-text"
  >
    SWAPI
    <img
      className="light-saber"
      src="light-saber.png"
    />
    BOX
  </h1>
</div>
`;
